hack/setup.sh

```diff
@@ -109,6 +109,26 @@ function captureGitToken {
   fi
 }
 
+function captureGitlabToken {
+   if [ -z "$GITLAB_TOKEN" ]; then
+    read -s -p "Enter Gitlab user access token: " value
+    echo ""
+    GITLAB_TOKEN=$value
+  else
+    echo "Gitlab access token already set."
+  fi
+}
+
+function captureGitlabHost {
```

Comment:
So I tried running this setup script (both with default and non default ) but it did not prompt me to  enter the GitLab token and host. Am I missing something other than the way we run it?